src/views/publish/dimension-chooser.ejs

```diff
@@ -18,7 +18,7 @@
         <% if (locals?.data) { %>
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-full with-overflow">
-                    <%- include("../partials/dimension-preview-table.ejs", { dimension: locals.dimension, headers: locals.headers, data: locals.data }) %>
+                    <%- include("../partials/view-table.ejs", { dimension: locals.dimension, headers: locals.headers, data: locals.data }) %>
```

Comment:
This file path does not exist in this branch (missing file commit?)

There's a `view-table.ejs` in `views/consumer/partials/` but not in `views/partials/`

Reply:
Fixed... This might have been a refactoring error